translations/hi.json

```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "हिंदी"
+  },
+  "panel": {
+    "config": "विन्यास",
```

Comment:
Let us remove this string. I am not sure most people will be able to understand what `विन्यास` means and many other translations don't have it.

Reply:
All coming from Google Translate... I am not a Hindi pudit you can tell :)

Reply:
Let's not use Google Translate. We should only get translations from native speakers.